src/utils/generator.js

```diff
@@ -881,6 +882,20 @@ export class Generator {
     return res;
   }
 
+  getModifiedBindings(): Set<{ modifiedBinding: Binding, oldValue: void | Value }> {
```

Comment:
Please add a comment that this might have different behavior than getting `generator.effects[2]` and that the `oldValue` in the Set returned should be agnostic to whether or not the generator's effects are applied.

Reply:
I will add comment and refactor code after our discussion.